Automattic/wp-calypso
# Add modern color scheme from wp-admin

One part of #47036

#### Changes proposed in this Pull Request

* Add modern color scheme

#### Testing instructions

* Visit /me/account
* Choose modern and save

To see the wp-admin color scheme, create an atomic site, then visit /wp-admin/profile.php, choose color scheme and save.

Calypso:
![2020-11-06_14-06_1](https://user-images.githubusercontent.com/937354/98409897-4652fa80-2039-11eb-868e-85aeb8097dd8.png)

Wp-admin:
![2020-11-06_14-06](https://user-images.githubusercontent.com/937354/98409908-494deb00-2039-11eb-8d27-934ed372c6f8.png)

#### Known Shortcomings

Wp-Admin has different behavior on sidebar hover for menu vs submenu.  Calypso doesn't support this; one hover style applies to both menus and submenus.  I haven't found a place to fit the green text color in anywhere.
![modern-hover-2](https://user-images.githubusercontent.com/937354/98410206-d729d600-2039-11eb-8402-cc6dcec3ab85.gif)
